cmd/api/main.go

```diff
@@ -28,8 +30,13 @@ func main() {
 		serverDeps.NewRelicApp = app
 	}
 
+	DB := db.Init()
+	h := handlers.New(DB)
+
 	server := api.NewServer(cfg, serverDeps)
```

Comment:
If we provide the DB instance through the serverDeps, we'll be able to more easily mock it out for testing.

Reply:
The NewServer func can be responsible for setting up the handlers too.

Reply:
added db to the server dependency